R/get_gene_expression.R

```diff
@@ -82,20 +72,20 @@ get_gene_expression = function(these_samples_metadata,
                                all_genes = FALSE,
                                expression_data,
                                from_flatfile = TRUE,
-                               prioritize_rows_by){
+                               collapse_duplicates = TRUE){
   
   # check parameters
   if(!is.null(join_with)){
     stopifnot("`join_with` must be one of NULL, \"mrna\", \"genome\", or \"capture\"." = 
                 join_with %in% c("mrna", "genome", "capture"))
   }
-  
   stopifnot("You must provide one of `hugo_symbols` and `ensembl_gene_ids` while `all_genes` is FALSE. Instead, you can just set `all_genes` to TRUE." = 
               sum(!missing(hugo_symbols), !missing(ensembl_gene_ids), all_genes) == 1 )
-  
   stopifnot("You did not specify a valid engine. Please use one of \"read_tsv\", \"grep\", \"vroom\", or \"fread\"." = 
               engine %in% c("read_tsv", "grep", "vroom", "fread"))
-  
+  if(default_priority & !missing(prioritize_rows_by)){
```

Comment:
These three lines need to be removed. I think these were left over from the earlier implementation that was recently scrapped.